test/test.js

```diff
@@ -4,44 +4,11 @@ var url = require('url');
 var expect = require('chai').expect;
 var objectAssign = require('object-assign');
 var format = require('string-format');
-var rimraf = require("rimraf");
-var fs = require('fs');
 var common = require('../lib/common')
 
-/*rimraf('sitespeed-result', function(err) {
-    console.log("delete legacy test data success")
-});
-*/
-
-var path1 = __dirname + '/../../../config/budget-testcases.json';
-var path2 = __dirname + '/../config/budget-testcases.json';
-var config = process.env.BUDGET_TESTCASES;
-if (!config) {
-    if (fs.existsSync(path1)) {
```

Comment:
@kakazuo I'm surprised you removed all of the stuff with loading the config from a file? Could you explain? I was never opposed to that... the comment in the other pull request was that I didn't want Bluemix-specific logic in this module for retrieving Bluemix login cookies.